### PR TITLE
Fix leak in GdiplusStartup

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

File: tests/Makefile.am [hunk 1]
```diff
@@ -306,7 +306,9 @@ EXTRA_DIST =			\
 	$(testtexturebrush_SOURCES)	\
 	$(testtext_SOURCES)	\
 	$(testtiffcodec_SOURCES) \
-	$(testwmfcodec_SOURCES)
+	$(testwmfcodec_SOURCES) \
+	lsansuppressions.txt \
```
Comment (akoeplinger) [inline tests/Makefile.am]: this file is missing

File: tests/Makefile.am [hunk 2]
```diff
@@ -306,7 +306,9 @@ EXTRA_DIST =			\
...
+	$(NULL)
```
Comment (akoeplinger) [inline tests/Makefile.am]: what is this?

File: README.md [hunk 3]
```diff
@@ -60,6 +60,14 @@ To run the tests with Clang sanitizers, run the following command from the root
 	./autogen.sh --enable-asan
 	make check
 
+To run the unit tests with leak sanitizers, run the following command from the root of the repository:
+
+	./autogen.sh --enable-asan
+	export ASAN_OPTIONS=detect_leaks=1
+	export ASAN_OPTIONS=fast_unwind_on_malloc=0
```
Comment (akoeplinger) [inline README.md]: @hughbe do we need this when we compile with ` -fno-omit-frame-pointer`? according to the FAQ (https://github.com/google/sanitizers/wiki/AddressSanitizer#faq) one of them should be enough

Comment (hughbe) [reply README.md]: On my linux VM things only worked when both were set

Comment (filipnavara) [reply README.md]: How does the second `export` not override the value from the first one?

Comment (hughbe) [reply README.md]: it doesn't seem to do that on my computer!

Comment (akoeplinger) [reply README.md]: Ok well it doesn't hurt since it's just in the docs :)

**Pull request merged**